tasks/configure_nginx.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: copy ssl cert
+  copy:
+    content: "{{ DALITE_SSL_CERT }}"
+    dest: "/etc/ssl/private/ssl-cert.pem"
+    group: www-data
+    mode: "740"
+
+- name: copy ssl key
+  copy:
+    content: "{{ DALITE_SSL_KEY }}"
+    dest: "/etc/ssl/private/ssl-cert.key"
+    group: www-data
+    mode: "740"
```

Comment:
Is it the right group for the private key? I don't remember if nginx needs to access the key while running with `www-data`, or if he does that with its root credentials and/or `ssl-cert`? Allowing `www-data` to access it seem insecure